src/app/users/ask-question/ask-question.component.ts

```diff
@@ -9,10 +26,101 @@ import { Post } from '../../models/Post';
 })
 export class AskQuestionComponent implements OnInit {
 
-  constructor() { }
+  form: FormGroup;
+  visible = true;
+  selectable = true;
+  removable = true;
+  addOnBlur = true;
```

Comment:
these should probably be tag-specific rather than global options.  It's OK for now, just noting it.